linux/utils/enable_auto_login.yml

```diff
@@ -16,40 +16,43 @@
     msg: "Can not enable auto login when guest_os_display_manager is {{ guest_os_display_manager | default('') }}."
   when: >
     guest_os_display_manager is undefined or
-    guest_os_display_manager not in ["gdm", "xdm", "lightdm", "lxdm", "sddm"]
+    guest_os_display_manager not in ["gdm", "xdm", "lightdm", "lxdm", "slim"]
```

Comment:
“sddm” is removed?

Reply:
sddm was used by FreeBSD. Now it is not needed. Till now, sddm is not used by any Linux in our support list